### DISCUSSION TITLE
Tags from dev.to

### DISCUSSION THREAD

**Discussion by devopg** [forem/forem#16336] [Q&A] [answered]
hello i create site by forem for headhusting.

is open source tags from dev.to ? 
Where i can download tags ?

**Comment by citizen428** [comment, marked as answer]
There's an API endpoint for retrieving tags: https://developers.forem.com/api#operation/getTags. It orders the tags by popularity and is paginated, so the first few pages may be enough for you.

**Reply by devopg** [reply]
very cool!
